src/layers/layers.jl

```diff
@@ -0,0 +1,48 @@
+module Layers
```

Comment:
I'm not sure this really needs to be a separate module since it's kind of the main/only thing the package does apart from the convenience functions for building standard model architectures, and I don't really see a risk of any sort of namespace conflicts...